.grype.yaml

```diff
@@ -10,3 +10,7 @@ ignore:
   - vulnerability: CVE-2021-22570
     package:
       name: google.golang.org/protobuf
+  # Fixing this CVE requires updating K8s et al to incompatible versions.
+  - vulnerability: GHSA-r53h-jv2g-vpx6
```

Comment:
I guess the ID Grype is using is `GHSA-v53g-5gjp-272r`, that's what I see in the SARIF output.